src/Core/NETPortable/NetPortableProfileTable.cs

```diff
@@ -80,7 +94,7 @@ internal static bool HasCompatibleProfileWith(NetPortableProfile packageFramewor
             return false;
         }
 
-        private static NetPortableProfileCollection BuildPortableProfileCollection()
+        public static NetPortableProfileCollection BuildPortableProfileCollection()
```

Comment:
suggestion : if this isn't being called from another assembly, can we make it internal?